resources/nats-streaming/values.yaml

```diff
@@ -16,8 +16,8 @@ global:
   containerRegistry:
     path: eu.gcr.io/kyma-project
   event_bus:
-    dir: develop/
-    version: 5e4264e9
+    dir: ""
```

Comment:
In the other occurrences, you did not use the `""`, is it safe to use both styles?

Reply:
correct me if im wrong: "" == empty string , without the "" it evaluates to null... the result is the same

Reply:
Yes, sure, but for the sake of consistency, should we make all of them with the same value?
My concern was only if the template engine would panic if he found the empty string, but fine forget about it 👍

Reply:
The templating language does the right thing in both case: generate an emtpy string.

Reply:
The templating language does the right thing in both cases: generate an emtpy string.